.circleci/config.yml

```diff
@@ -0,0 +1,36 @@
+version: 2
+jobs:
+  build:
+    docker:
+      - image: circleci/rust:1.36.0
+    steps:
+      - checkout
+      - run:
+          name: setup
+          command: |
+            rustup component add rustfmt --toolchain 1.36.0-x86_64-unknown-linux-gnu
+            rustup install nightly
+      - run:
+          name: cargo build
+          command: |
+            cargo --version --verbose
+            cargo build
+      - run:
+          name: cargo fmt
+          command: |
+            cargo fmt --version
+            cargo fmt -- --check
+      - run:
+          name: cargo test
+          command: |
+            cargo test
```

Comment:
add `--locked` to ensure lock file is update-to-date
https://github.com/cennznet/cennznet/blob/db8193885cac3908455a3951e155537e58e5fccc/.circleci/config.yml#L74